depends/packages/rapidcheck.mk

```diff
@@ -0,0 +1,21 @@
+package=rapidcheck
+
+$(package)_version=1.0
+
+$(package)_download_path=https://github.com/Christewart/rapidcheck/releases/download/1.0
+
+$(package)_file_name=$(package)-$($(package)_version).tar.gz
+
+$(package)_sha256_hash=c228dc21ec24618bfb6afa31d622d1f4ea71168f04ee499e1ffcfc63cd5833f4
```

Comment:
Can you try to adjust the folder name within the targz to have the version appened?

This may fix the travis issue.